server/adaptors/integrations/__data__/repository/haproxy/info/test/docker-compose.yaml

```diff
@@ -0,0 +1,118 @@
+version: '3'
+
+services:
+  opensearch:
+    image: opensearchproject/opensearch:2.11.0
+    container_name: opensearch
+    environment:
+      - cluster.name=opensearch-cluster # Name the cluster
+      - node.name=opensearch # Name the node that will run in this container
+      - discovery.seed_hosts=opensearch # Nodes to look for when discovering the cluster
+      - cluster.initial_cluster_manager_nodes=opensearch # Nodes eligibile to serve as cluster manager
+      - bootstrap.memory_lock=true # Disable JVM heap memory swapping
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # Set min and max JVM heap sizes to at least 50% of system RAM
+      - "DISABLE_INSTALL_DEMO_CONFIG=true" # Prevents execution of bundled demo script which installs demo certificates and security configurations to OpenSearch
+      - "DISABLE_SECURITY_PLUGIN=true" # Disables security plugin
+    ulimits:
+      memlock:
+        soft: -1 # Set memlock to unlimited (no soft or hard limit)
+        hard: -1
+      nofile:
+        soft: 65536 # Maximum number of open files for the opensearch user - set to at least 65536
+        hard: 65536
+    volumes:
+      - opensearch:/usr/share/opensearch/data # Creates volume called opensearch-data1 and mounts it to the container
+    ports:
+      - 9200:9200
+      - 9600:9600
+    expose:
+      - "9200"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9200/_cluster/health?wait_for_status=yellow"]
+      interval: 5s
+      timeout: 25s
+      retries: 4
+    networks:
+      - opensearch-net # All of the containers will join the same Docker bridge network
+  opensearch-dashboards:
+    image: pothulakalyanireddy/dashboard-app:dev
```

Comment:
In general I'd like to avoid relying on external accounts for test images, would it be possible to use e.g. the Dockerfile in the repository root to set up an instance? Alternatively we can remove this section entirely and assume that the OSD instance is run on the outside and connecting to the OS instance here, that's how I usually do development with integrations.

For version compatibility with something based on this branch,  the opensearch image would also need to be `opensearchstaging/opensearch:3.0.0`.

Reply:
Hi, with this version of opensearchstaging/opensearch:3.0.0 the dashboard version (2.11.0)  opensearchproject/opensearch-dashboards:latest is incompatible facing issue of (OpenSearch dashboard server is not yet ready) 
. I tried using opensearchstaging/opensearch-dashboards:3.0.0 in this I could not find integrations. 
![image](https://github.com/opensearch-project/dashboards-observability/assets/98814079/05d994b9-87da-45ce-801e-1aa3ef693513)

Reply:
It's because the staging image doesn't include plugins, I'm not sure if we have a widely-used method to add a dashboard image that has some plugins installed. I think a good route is to just remove the dashboards image from this compose file and let the user set it up externally.

Reply:
Thank you so much for the support.